values.yaml

```diff
@@ -498,6 +498,11 @@ st2chatops:
     requests:
       memory: "50Mi"
       cpu: "5m"
+  # Advanced use-cases only. If defined, this hubot-scripts volume gets mounted to: /opt/stackstorm/chatops/scripts
+  # This volume (using any k8s storage solution, including configmap) allows for hubot customization.
+  # Most installations should not use this.
+  # Please refer to the hubot documentaiton for details on writing .js or .coffeescript hubot extensions.
+  hubotScriptsVolume: {}
```

Comment:
I think it worth mentioning https://github.com/hubot-scripts here.
So users will have a better idea about the custom hubot scripts.

Reply:
How about overwriting the `external-scripts.json`?
That's the config you mentioned users would need to enable the external hubot scripts thy provided via volume.

Reply:
You only have to add things to `external-scripts.json` if they are *not* in the `scripts/` directory. So, we have to add `stackstorm-hubot` because it is in node_modules. When I drop my coffeescript file in `scripts/`, I do not register it in `external-scripts.json`.

I'll add the docs link soon.

Reply:
If you want to make `external-scripts.json` updatable, I would probably turn it into a configmap template so we can ensure hubot-stackstorm is registered. Would you like me to do that?

Reply:
OK. I made `external-scripts.json` updatable.

You still have to update the container though to install such scripts, so you could just edit `external-scripts.json` in the container itself. But I'm indifferent on external-scripts.json, because I don't need to touch it.

Reply:
> You only have to add things to external-scripts.json if they are not in the scripts/ directory. So, we have to add stackstorm-hubot because it is in node_modules. When I drop my coffeescript file in scripts/, I do not register it in external-scripts.json.

Makes sense, thanks for more details, I didn't know :+1: 

If what's included in the `scripts/` dir is already picked by the system, it's safer to revert that `external-scripts.json` customization as it has no play. Definitely would avoid shooting users in their foot.

Sorry for the confusion there.

Reply:
K, I dropped that change. This is ready now.